### ISSUE TITLE
Handle DESIRED_usage_model=OPPORTUNISTIC,DEDICATED for backward compatibility

### ISSUE THREAD

**Issue by shreyb** [marcmengel/jobsub_lite#5]
Put something in jobsub lite where if someone said DESIRED_usage_model=OPPORTUNISTIC,DEDICATED on a jobsub_submit line, it would trigger DESIRED_Sites="Fermigrid".   This value should be overrideable by an environment variable.

Note that this is a stopgap for backward compatibility, and is not something we want to encourage.  Once this is done, we need to open another issue to plan its phase-out, along with a possible replacement (maybe an --onsite flag that's configurable in the environment?)

**Comment by goodenou** [comment]
Fixed in PR #101
